share/picongpu/tests/Synchrotron_plugin/bin/validate.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2024 PIConGPU contributors
+# Authors: Mika Soren Voss, Rene Widera, Hannes Wolf, Klaus Steiniger, Filip Optolowicz
+# License: GPLv3+
+#
+
+set -o pipefail
+
+# define testSuite variable, set correctly further below if
+# buildsystem/CompileSuite/color.sh from PIConGPU is available
+testSuite="" # define variable, set correctly
+
+currentDir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+picongpuPrefix=""
+
+if [ ! -z $PICSRC ]; then
+    picongpuPrefix="$PICSRC"
+elif [ ! -z $CI_PROJECT_DIR ]; then
+    picongpuPrefix="$CI_PROJECT_DIR"
+fi
+
+printf "PIConGPU prefix = %s\n" $picongpuPrefix
+
+## Load libs and functions to use the thumbs_up and thumbs_down
+if [ ! -z $picongpuPrefix ]; then
+    if [ -d $picongpuPrefix ]; then
+        printf "Load buildsystem/CompileSuite/color.sh from PIConGPU!\n"
+        . $picongpuPrefix/buildsystem/CompileSuite/color.sh
+        testSuite=`echo_b "[testSuite] "`
```

Comment:
```suggestion
        testSuite=$(echo_b "[testSuite] ")
```
Do this everywhere because of [this](https://www.redhat.com/sysadmin/backtick-operator-vs-parens).

Reply:
Changed in two places